binding/flutter/ios/Classes/SwiftPorcupinePlugin.swift

```diff
@@ -11,9 +11,150 @@
 
 import Flutter
 import UIKit
+import Porcupine
+
+enum Method : String {
+    case INIT
+    case CREATE
+    case PROCESS
+    case DELETE
+    case DEFAULT
+}
 
 public class SwiftPorcupinePlugin: NSObject, FlutterPlugin {
-  public static func register(with registrar: FlutterPluginRegistrar) {
+    private var porcupinePool:Dictionary<String, Porcupine> = [:]
+    
+    public static func register(with registrar: FlutterPluginRegistrar) {
+        let instance = SwiftPorcupinePlugin()
+
+        let methodChannel = FlutterMethodChannel(name: "porcupine", binaryMessenger: registrar.messenger())
+        registrar.addMethodCallDelegate(instance, channel: methodChannel)
+    }
+    
+    public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+        let method = Method(rawValue: call.method.uppercased()) ?? Method.DEFAULT
```

Comment:
again, maybe no default is necessary. Just return an error it you get a weird method call